src/rws_client.cpp

```diff
@@ -478,7 +478,9 @@ RWSResult RWSClient::setSpeedRatio(unsigned int ratio)
 RWSResult RWSClient::loadModuleIntoTask(const std::string& task, const FileResource& resource, const bool replace)
 {
   std::string uri = generateRAPIDTasksPath(task) + "?" + Queries::ACTION_LOAD_MODULE;
-  std::string content = Identifiers::MODULEPATH + "=" + generateFilePath(resource) + "&replace=" + ((replace) ? "true" : "false");
+  std::string content =
+      Identifiers::MODULEPATH + "=" + resource.directory + "/" + resource.filename +
+      "&replace=" + ((replace) ? "true" : "false");
```

Comment:
What happens here? What is the difference?

Reply:
Earlier it was using `generateFilePath(resource)` which produced `Services::FILESERVICE + "/" + resource.directory + "/" + resource.filename`, but in this request there shouldn't be `Services::FILESERVICE + "/"` part.